features/frameworks/utest/source/utest_harness.cpp

```diff
@@ -47,8 +47,15 @@ namespace
     size_t case_failed = 0;
     size_t case_failed_before = 0;
 
-    handlers_t defaults = default_handlers;
-    handlers_t handlers = defaults;
+    handlers_t& defaults() {
```

Comment:
> On the other hand, it can be worked around by creating a new class derived from handlers_t which default initialize to default_handlers . Do you prefer this solution ?

That would be great!